lib/src/booleans/boolean_contains.dart

```diff
@@ -0,0 +1,481 @@
+import 'package:turf/turf.dart';
+
+import 'boolean_point_in_polygon.dart';
+import 'boolean_point_on_line.dart';
+
+/// Boolean-contains returns True if the second geometry is completely contained
+/// by the first geometry.
+/// The interiors of both geometries must intersect and, the interior and
+/// boundary of the secondary must not intersect the exterior of the primary.
+/// Boolean-contains returns the exact opposite result of the `turf/boolean-within`.
+/// example:
+/// ```dart
+/// var line = LineString(coordinates: [Position.of([1, 1]), Position.of([1, 2]), Position.of([1, 3]), Position.of([1, 4])]);
+/// var point = Point(cooridantes:Position.of([1, 2]));
+/// booleanContains(line, point);
+/// //=true
+/// ```
+booleanContains(GeoJSONObject feature1, GeoJSONObject feature2) {
+  var geom1 = feature1 is Feature ? feature1.geometry : feature1;
+  var geom2 = feature2 is Feature ? feature2.geometry : feature2;
+  var type1 = geom1!.type;
+  var type2 = geom2!.type;
+  var coords1 = (geom1 as GeometryType).coordinates;
+  var coords2 = (geom2 as GeometryType).coordinates;
+  switch (type1) {
+    case GeoJSONObjectType.point:
+      switch (type2) {
+        case GeoJSONObjectType.point:
+          return compareCoords(coords1, coords2);
+        default:
+          throw UnsupportedError(
+              "{feature2   $type2   geometry not supported}");
+      }
+    case GeoJSONObjectType.multiPoint:
+      switch (type2) {
+        case GeoJSONObjectType.point:
+          return isPointInMultiPoint(geom1 as MultiPoint, geom2 as Point);
+        case GeoJSONObjectType.multiPoint:
+          return isMultiPointInMultiPoint(
+              geom1 as MultiPoint, geom2 as MultiPoint);
+        default:
+          throw UnsupportedError(
+              "{feature2   $type2   geometry not supported}");
```

Comment:
Use if-statements here with typechecks `type is MultiPoint` e.g. 

This way we can remove the unsave type casting at the beginning of the function and all of the `as` operators